generated/nidaqmx/_task_modules/triggering/start_trigger.py

```diff
@@ -1037,6 +1037,20 @@ def cfg_dig_pattern_start_trig(
         self._interpreter.cfg_dig_pattern_start_trig(
             self._handle, trigger_source, trigger_pattern, trigger_when.value)
 
+    def cfg_time_start_trig(self, when, timescale=Timescale.USE_HOST):
+        """
+        New Start Trigger
```

Comment:
The first line of a docstring should be a summary. We should figure out why the metadata contains "New Start Trigger" for the description.

https://peps.python.org/pep-0257/

Reply:
This is due to the patch file in nidmxa having incorrect documentation. I have posted a PR on that here https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/640231